sql/src/main/java/io/crate/execution/engine/collect/files/FileReadingIterator.java

```diff
@@ -151,7 +143,8 @@ private void initCollectorState() {
                 FileInput fileInput = getFileInput(fileUri.uri);
                 fileInputs.add(new Tuple<>(fileInput, fileUri));
             } catch (IOException e) {
-                rethrowUnchecked(e);
+                lineProcessor.startWithUri(fileUri.uri);
```

Comment:
yes but it's wrong, not working (no row is emitted which would read the exception, so it would never result in an error summary entry) and senseless ;-)  as actually no `FileInputFactory` implementation will throw an `IOException`. I'll adjust the interface and remove this...